examples/snowflake/.saturn/saturn.json

```diff
@@ -0,0 +1,22 @@
+{
+    "image": "saturncloud/saturn-rapids:2021.02.22",
+    "jupyter": {
+        "size": "g4dnxlarge",
+        "disk_space": "10Gi",
+        "ssh_enabled": false
+    },
+    "dask_cluster": {
+        "n_workers": 3,
+        "nthreads": 1,
+        "scheduler_size": "medium",
+        "worker_size": "g4dnxlarge"
+    },
+    "environment_variables": {
+        "TAXI_DATABASE": "saturn_nyc_taxi",
+        "JUPYTERLAB_SATURN_EXTENSION_DISABLED": "True"
+    },
+    "description": "Connect to a Snowflake database from either a single machine or distributed clsuter",
+    "title": "Snowflake",
+    "thumbnail_image_url": "https://saturn-public-assets.s3.us-east-2.amazonaws.com/example-thumbnails/snowflake.png",
+    "weight": 2
```

Comment:
I am going to open a PR to enforce uniqueness among weights. So it'd be good to have them not all be 2 :). I think this one should be fairly high. Maybe 10 for instance.

Reply:
I actually didn't realize how weights worked - I thought it was more a rank order thing and ties would be chosen some other way. I'll update :)

Reply:
No that is how they work. But we should know what order we want for these things and set them intentionally.